great_expectations/dataset/__init__.py

```diff
@@ -1,3 +1,4 @@
-from .base import Dataset
+from .base import Dataset, DataTable, DataFile
 from .pandas_dataset import MetaPandasDataset, PandasDataset
```

Comment:
I think we want to rename more comprehensively, so MetaPandasDataset --> MetaPandasDatatable (etc), keeping with the table name. I'm going to put this in slack for another final checkoff by all.